packages/building-rollup/modern-and-legacy-config.js

```diff
@@ -119,7 +119,7 @@ function createConfig(_options, legacy) {
       entrypointHashmanifest(),
 
       production &&
-        options.plugins.workbox &&
+        options.plugins.workbox && !legacy &&
```

Comment:
```suggestion
        options.plugins.workbox && 
        !legacy &&
```

Because prettier is complaining ^^